.github/workflows/docs.yml

```diff
@@ -0,0 +1,19 @@
+name: Docs
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+jobs:
+  verify:
+    runs-on: [self-hosted]
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Run check format
+        run: npm run check:format
```

Comment:
Does it need to install any dev dependencies, if so, add a step that installs the dependencies.